schema/item_fr.py

```diff
@@ -0,0 +1 @@
+item_en.py
```

Comment:
what would happen if we enable this in the unpacker?

Reply:
github doesn't show this well (or at all?), but the file is a symlink. On linux it works perfectly, but I have no idea what's the behaviour on windows.

Reply:
hmm I wonder what would happen if I create an .exe then run it

Reply:
Apparently newer versions of git can create symlinks on windows as well with a certain configuration option (core.symlinks = true). You could try that, or I could simply duplicate the schema file.

Reply:
yeah it just shows item_en.py when I pull it. So probably just duplicate it

Reply:
Done.